branch-to-env/parseEnvironmentName.js

```diff
@@ -0,0 +1,28 @@
+function parseBranch (ref) {
+  const regex = /refs\/(heads|tags)\/(\S*)/
```

Comment:
so it just doing `echo ${GITHUB_REF#refs/heads/}`. 

```bash
$ GITHUB_REF=refs/heads/feature-branch-1
$ echo ${GITHUB_REF##*/}
feature-branch-1
```

also, if it about branches, why here also tags?

Reply:
but we also map the branch/tag names to environments/deploy targets @le0pard, that's the main job of this action.

it can be done with a combination of existing tools, but we decided to have this action to reduce duplication (of which there's already too much in GH actions)

Reply:
In this case it is cannot be called `parseBranch`, isn't it? Because it also parse tags @leonid-shevtsov

Reply:
@le0pard  maybe `githubRefToEnv` ?

Reply:
@IgorDobryn yep, will work